libs/designsystem/src/lib/components/form-field/form-field.component.spec.ts

```diff
@@ -518,4 +520,80 @@ describe('FormFieldComponent', () => {
       expect(secondEvent.type).toBe('touchend');
     });
   });
+  describe('affix', () => {
+    const wait = (ms: number = 25): Promise<void> => {
+      return new Promise((resolve) => {
+        setTimeout(resolve, ms);
+      });
+    };
+    describe('with prefix', () => {
+      beforeEach(() => {
+        spectator = createHost(
+          `<kirby-form-field>
+            <span kirby-affix="prefix" style="width: 50px">foo</span>
+            <input kirby-input />
+           </kirby-form-field>`
+        );
+      });
+      it('should render prefix content', () => {
+        const affix = spectator.query(byText('foo'));
+        expect(affix).toBeTruthy();
+      });
+      it('should render prefix content in correct slot', () => {
+        const affix = spectator.query(byText('foo'));
+        expect(affix.parentElement.classList).toContain('prefix');
+      });
+      it('should add width of prefix content to padding-left of <input>', async () => {
+        await wait();
+        const input = spectator.queryHost('input');
+        const expectedPadding = 50 + parseInt(DesignTokenHelper.size('s'));
+        expect(input).toHaveComputedStyle({
+          'padding-left': `${expectedPadding}px`,
+        });
+      });
+    });
+    describe('with suffix', () => {
+      beforeEach(() => {
+        spectator = createHost(
+          `<kirby-form-field>
+            <span kirby-affix="suffix" style="width: 50px">foo</span>
+            <input kirby-input />
+           </kirby-form-field>`
+        );
+      });
+      it('should render suffix content', () => {
+        const affix = spectator.query(byText('foo'));
+        expect(affix).toBeTruthy();
+      });
+      it('should render suffix content in correct slot', () => {
+        const affix = spectator.query(byText('foo'));
+        expect(affix.parentElement.classList).toContain('suffix');
+      });
+      it('should add width of suffix content to padding-right of <input>', async () => {
+        await wait();
```

Comment:
Without the wait function the `input` field seemed to never have the style applied to it. It seems like a hack but I couldn't figure out any other way to test for the padding.

Reply:
I would prefer to leave out these tests, as I cannot figure out a good way to wait for both the ngAfterContentInit hook and the ResizeObserver to have completed before we check the styles. 

Have tried any sensible combination of our TestHelper methods without luck, so instead I pushed a new version that just deletes these (otherwise really sane!) tests, so we can get on with it.